### ISSUE TITLE
Users connecting/disconnecting doesn't get updated in Nostalgia and Obsession

### ISSUE THREAD

**Issue by Knezzen** [jhalter/mobius#84]
Hi!

Happy holidays :)

I recently noticed that users connecting and disconnecting didn't get updated properly when using Nostalgia and Obsession.
It works when using hxd and the official Hotline server, so I'm thinking this probably is a Mobius issue.

People disconnecting keep being in the user list on Nostalgia and Obsession and only a disconnect and reconnect to the server updates the user list to display the correct amount of connected users.

**Comment by jhalter** [comment]
Hi there, happy holidays! ☃️

I'll take a look in the next few days.

**Comment by jhalter** [comment]
Hi, I've released v0.10.12 with a fix for the  Obsession client.

I wasn't able to reproduce the problem in the Nostalgia client, at least in version 0.991.

**Comment by Knezzen** [comment]
I have installed 0.10.12 and it seems to fix the problem, but I have observed it on other clients as well.
Maybe they are a bit odd in terms of how many people actually use them, but they are still a bit incompatible with Mobius.

Tested with Pitbull Pro 2.4.4 on Mac OS Classic and with Heidrun on Mac OS X Tiger and they have the same issue as Obsession had. Is there any other way to determine client compatibility?

I also noticed that the threaded news section doesn't display what's actually in the news item when using Frogblast, but that's for another ticket :)

Thanks for working on this wonderful piece of software! :)
